WordPress/Classes/ViewRelated/Jetpack/Branding/Banner/JetpackBannerView.swift

```diff
@@ -38,7 +38,7 @@ class JetpackBannerView: UIView {
     }
 
     /// Preferred minimum height to be used for constraints
-    static let minimumHeight: CGFloat = 50
+    static let minimumHeight: CGFloat = 44
```

Comment:
Awesome! Looks like we've done a good job making this the source of truth. I tested all views I'm aware of with a banner and they appear to correctly reflect the change.